x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -338,3 +339,8 @@ joinCondition
 joinPredicate
     : valueExpression
     ;
+
+insistCommand
+    : DEV_INSIST identifier
```

Comment:
This being an identifier makes `insist some.subfield` fail on the existence of the dot as a parsing exception. Whereas something like `rename some.subfield as foo` works. `insist` would only work with dotted field names if they are surrounded by backticks which will be an inconsistent behavior with commands that also use field names.